packages/web-frontend/src/components/View/MatrixWrapper/MatrixWrapper.js

```diff
@@ -155,15 +155,11 @@ export class MatrixWrapper extends Component {
     this.updateWrapper = this.updateWrapper.bind(this);
   }
 
-  componentDidMount() {
-    const { startDate, endDate } = this.props.viewContent || {};
-    this.props.onSetDateRange(startDate, endDate); // Force a load when the expanded matrix first mounts
-  }
-
```

Comment:
This was the only change. The matrix will always load on mount (initiated in `EnlargedDialog`)